dev/com.ibm.ws.microprofile.config_fat/test-applications/cdiConfig.jar/src/com/ibm/ws/microprofile/appConfig/cdi/web/SessionScopedFieldTestServlet.java

```diff
@@ -17,7 +17,7 @@
 
 @SuppressWarnings("serial")
 @WebServlet("/fieldSession")
-public class SessionScopedFieldTestServlet extends AbstractBeanServlet {
+public class SessionScopedFieldTestServlet extends FieldTestServlet {
```

Comment:
This doesn't seem to be used any more. Can it be removed?

Reply:
done